### DISCUSSION TITLE
Lock Pinned Tabs to Stay within a Specific Domain

### DISCUSSION THREAD

**Discussion by trlinden** [zen-browser/desktop#2750] [Ideas]
Is there any way to get the following functionality?

I have Outlook webmail opened as a pinned tab. I would like to keep it set to check my e-mail, but would like to be able to click around within outlook to send mails, look at different inboxes etc.

However, if I click a link in an e-mail that takes me to a non-outlook site (outside of the outlook domain), I would like it to open that external link in a new tab (and immediately go to that new tab) - so that the pinned tab stays locked within the outlook domain (and I can continue to get e-mail notifications, etc.)

This is potentially useful for all sorts of things, like gmail, whatsapp chats, AI assistants, etc. It mirrors the behavior you would expect if outlook was a different program, that was opening a www link in a web browser (which is intuitive for most people).

Is there any way this is possible?

**Comment by 12th-devs** [comment]
Not that I know of, sounds like a great feature though!

**Comment by hb-robo** [comment]
+1. I think without this feature, the pinned tabs are a bit unwieldy.

Furthermore, I think search bar searches or typed website links of any kind should also break out into its own new, unpinned tab. Pinned tabs need to remain in their proper context, otherwise they become a bit of a mess.
